src/vulnpy/trigger/nosqli.py

```diff
@@ -0,0 +1,48 @@
+from pymongo import MongoClient
+import os
+
+
+def create_mongo_client(**kwargs):
+    mongo_host = [os.environ.get("MONGODB_HOST", "localhost:27017")]
+    return MongoClient(host=mongo_host, **kwargs)
+
+
+client = create_mongo_client()
+db = client.pymongo_test_db
+
+
+def do_nosqli_find(user_input):
+    data = [x for x in db.spam.find({"title": user_input})]
+    return data
+
+
+def do_nosqli_insert_one(user_input):
+    db.posts.insert_one({"key_one": "value_one"})
+    db.posts.insert_one({"title": user_input})
+    data = db.posts.find_one({"key_one": "value_one"})
+    return data["key_one"]
+
+
+def do_nosqli_insert_many(user_input):
+    db.posts.insert_one({"key_one": "value_one"})
+    db.posts.insert_many([{"title": user_input}])
+    data = db.posts.find_one({"key_one": "value_one"})
+    return data["key_one"]
+
+
+def do_nosqli_update(user_input):
+    db.posts.insert_one({"key_one": "value_one"})
+    record = {"title": "Old title", "content": "PyMongo is fun!", "author": "Dani"}
+    db.posts.insert_one(record)
+    db.posts.update_one(record, {"$set": {"title": user_input}})
```

Comment:
many (maybe all?) of the pymongo vulnerable functions here return a value, and I'd prefer if the return value from our trigger function contained at least some representation of that value. Right now, some of these returns are a little misleading because they look up records that aren't related to user_input. I understand this could be a little extra work, but I think it would be worth the effort